client/lib/plans/constants.js

```diff
@@ -437,6 +447,22 @@ export const featuresList = {
 		),
 		plans: allPaidPlans
 	},
+	[ FEATURE_STANDARD_SECURITY_TOOLS ]: {
+		getTitle: () => i18n.translate( 'Standard Security Tools' ),
+		getDescription: () => i18n.translate( 'Brute force protection, uptime monitoring, secure sign on, and automatic updates for your plugins.' )
+	},
+	[ FEATURE_SITE_STATS ]: {
+		getTitle: () => i18n.translate( 'Site Stats and Analytics' ),
+		getDescription: () => i18n.translate( 'The most important metrics for your site.' )
+	},
+	[ FEATURE_TRAFFIC_TOOLS ]: {
+		getTitle: () => i18n.translate( 'Traffic and Promotion Tools' ),
+		getDescription: () => i18n.translate( 'Build and engage your audience with more than a dozen optimization tools.' )
+	},
+	[ FEATURE_MANAGE ]: {
+		getTitle: () => i18n.translate( 'Centralized Dashboard' ),
+		getDescription: () => i18n.translate( 'Manage all of your WordPress sites from one location.' )
```

Comment:
Could you add `getSlug` method to these please? Like this: https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/constants.js#L441. It's currently used for feature highlighting added in this PR: https://github.com/Automattic/wp-calypso/pull/7188.